src/internal/pjsdb/queue_crud.go

```diff
@@ -62,3 +64,43 @@ func DequeueAndProcess(ctx context.Context, tx *pachsql.Tx, programHash []byte)
 	}
 	return resp, nil
 }
+
+type GetQueueResponse struct {
```

Comment:
Hey is there a `Queue` Struct? If so, what fields does it have? I'm thinking this response should be the Queue struct. But if that's too hard to accomplish, then we can figure that out later.